content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,184 @@
+mengkonfigurasi hal ini dengan opsi `--max-endpoints-per-slice` {{< glossary_tooltip
+text="kube-controller-manager" term_id="kube-controller-manager" >}} sampai dengan
+jumlah maksimum sebanyak 1000 _endpoint_.
+
+### Distribusi EndpointSlice
+
+Tiap EndpointSlice memiliki serangkaian _port_ yang berlaku untuk seluruh _endpoint_ dalam sebuah sumber daya. Ketika nama _port_ digunakan untuk sebuah Service, Pod mungkin mendapatkan
+nomor target _port_ yang berbeda-beda untuk nama _port_ yang sama, sehingga membutuhkan
+EndpointSlice yang berbeda. Hal ini mirip dengan logika mengenai bagaimana _subset_ dikelompokkan
+dengan Endpoints.
+
+_Controller EndpointSlice_ akan mencoba untuk mengisi EndpointSlice sebanyak mungkin, tetapi tidak
+secara aktif melakukan _rebalance_ terhadap EndpointSlice tersebut. Logika dari _controller_ cukup sederhana:
+
+1. Melakukan iterasi terhadap EndpointSlice yang sudah ada, menghapus _endpoint_ yang sudah tidak lagi
+   dibutuhkan dan memperbarui _endpoint_ yang sesuai yang mungkin telah berubah.
+2. Melakukan iterasi terhadap EndpointSlice yang sudah dimodifikasi pada langkah pertama dan
+   mengisinya dengan _endpoint_ baru yang dibutuhkan.
+3. Jika masih tersisa _endpoint_ baru untuk ditambahkan, mencoba untuk menambahkannya pada
+   _slice_ yang tidak berubah sebelumnya dan/atau membuat _slice_ yang baru.
+   
+Terlebih penting, langkah ketiga memprioritaskan untuk membatasi pembaruan EndpointSlice terhadap
+distribusi dari EndpointSlice yang benar-benar penuh. Sebagai contoh, jika ada 10
+_endpoint_ baru untuk ditambahkan dan ada 2 EndpointSlice yang masing-masing memiliki ruang untuk 5 _endpoint_ baru,
+pendekatan ini akan membuat sebuah EndpointSlice baru daripada mengisi 2
+EndpointSlice yang sudah ada. Dengan kata lain, pembuatan sebuah EndpointSlice 
+lebih diutamakan daripada pembaruan beberapa EndpointSlice.
+
+Dengan kube-proxy yang berjalan pada tiap Node dan mengamati EndpointSlice, setiap perubahan
+pada sebuah EndpointSlice menjadi sangat mahal karena hal tersebut akan dikirimkan ke
+setiap Node dalam klaster. Pendekatan ini ditujukan untuk membatasi jumlah
+perubahan yang perlu dikirimkan ke setiap Node, meskipun hal tersebut berdampak pada banyaknya
+EndpointSlice yang tidak penuh.
+
+Pada praktiknya, distribusi yang kurang ideal seperti ini akan jarang ditemukan. Kebanyakan perubahan yang diproses oleh _controller_ EndpointSlice akan sangat kecil untuk dapat masuk pada
+EndpointSlice yang sudah ada, dan jika tidak, cepat atau lambat sebuah EndpointSlice baru 
+akan segera dibutuhkan. _Rolling update_ dari Deployment juga menyediakan sebuah proses
+_repacking_ EndpointSlice yang natural seiring dengan digantikannya seluruh Pod dan _endpoint_ yang 
+bersangkutan.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* [Mengaktifkan EndpointSlices](/docs/tasks/administer-cluster/enabling-endpointslices)
```

Comment:
EndpointSlices could be EndpointSlice-EndpointSlice, perhaps?

Reply:
Ah, I think that one would be better with singular form `EndpointSlice`. Plural terms have been treated the same way in other translated pages.

Reply:
I'll provide more rationale @sftim 

IMO `EndpointSlice-EndpointSlice` in this context will sound weird and distract/confuse the reader.
The problem with translating plural words from English to Indonesian is because Indonesian doesn't always convey plurality directly in the word itself (by using the "repeated form"). In English, just adding 's/es' to words will embed plurality into the word. In Indonesian, it depends on the context of the sentence. For example:

1. `Look at those cats!` would be translated into `Lihat kucing-kucing itu!`
2. `There are so many cats!` would be translated into `Ada banyak kucing!`, the plural is already implied in the word "many/banyak".
3. `Cats have 9 lives` would be translated into `kucing memiliki 9 nyawa`. The context is talking about cats in general, which already imply plural cats; the plurality isn't even important in Indonesian in this context.

Here, we have the word `EndpointSlices`. This is a technical word in Kubernetes, referring to the API Object. It takes its name from a compound word, `endpoint + slices` (endpoint is even a compound word) which implies the meaning "slice of endpoint(s)".
Now we have another challenge of trying to translate a compound word in plural form from English to Indonesian (which is another topic worth discussing, but probably not here). This would be easy, we can see `EndpointSlice` literally as an API Object, so we can just translate `EndpointSlices` into `EndpointSlice-EndpointSlice`. But here, we use the "repeated word" plural form, which actually emphasizes a feel of plurality in Indonesian in which **there are the actual objects being talked about, being present in the context**.

In this context, the doc is pointing the user to another doc about enabling the EndpointSlice **feature**, not the EndpointSlice objects. Trying to translate it into `EndpointSlice-EndpointSlice` will generally make the readers think that they are going to an article on **how they are going to create the objects**, not how to enable the feature. 
Now here as the doc is talking about how to enable the `EndpointSlice` feature in general, the plurality is not really important for Indonesian readers, so using the singular `EndpointSlice` will be better to avoid reader confusion.

Reply:
> using the singular EndpointSlice will be better to avoid reader confusion.

Makes sense to me.